zou/cli.py

```diff
@@ -136,26 +136,31 @@ def create_admin(email, password):
     Create an admin user to allow usage of the API when database is empty.
     """
     try:
-        auth.validate_password(password)
-        # Allow "admin@example.com" to be invalid.
-        if email != "admin@example.com":
-            auth.validate_email(email)
-        password = auth.encrypt_password(password)
-        persons_service.create_person(
-            email, password, "Super", "Admin", role="admin"
-        )
-        print("Admin successfully created.")
-
-    except IntegrityError:
-        print("User already exists for this email.")
-        sys.exit(1)
-
-    except auth.PasswordTooShortException:
-        print("Password is too short.")
-        sys.exit(1)
-    except auth.EmailNotValidException:
-        print("Email is not valid.")
-        sys.exit(1)
+        person = persons_service.get_person_by_email(email)
+        if person["role"] != "admin":
+            persons_service.update_person(person["id"], {"role": "admin"})
```

Comment:
Can you write a message in the console telling that the user was upgraded?

Reply:
Done.
I realized when re-reading the code that there was still a `except IntegrityError` that might become obsolete with this modification.
Do you know if there some other reason that this exception could be raised or should I just remove it ?

Reply:
I think it is raised by SQL alchemy when the entry is already present. So yes, you can remove it!

Reply:
Alright, it's done.